src/main/java/org/broadinstitute/hellbender/engine/FeatureDataSource.java

```diff
@@ -253,6 +253,11 @@ public FeatureDataSource(final FeatureInput<T> featureInput, final int queryLook
         } else {
             throw new GATKException("Found a feature input that was neither GenomicsDB or a Tribble AbstractFeatureReader.  Input was " + featureInput.toString() + ".");
         }
+        // Due to a bug in HTSJDK, unindexed block compressed input files may fail to parse completely. For safety,
+        // these files have been disabled. See https://github.com/broadinstitute/gatk/issues/4224 for discussion
+        if (!hasIndex && TribbleIndexedFeatureReader.hasBlockCompressedExtension(featureInput.getFeaturePath())) {
+            throw new UserException.MissingIndex("Found a feature input that was block compressed but lacked an index file, support for unindexed feature files has temporarily been removed.  Input was " + featureInput.toString() + ".");
```

Comment:
Include instructions on how to index the file in the error message ("You can use the IndexFeatureFile tool to index this file")

Reply:
done